packages/kinetic-client/kinetic-client.0.0.6/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+name: "kinetic-client"
+version: "0.0.6"
+maintainer: "Romain Slootmaekers <romain.slootmaekers@cloudfounders.com>"
+authors: "Romain Slootmaekers <romain.slootmaekers@cloudfounders.com>"
+homepage: "https://github.com/cloudfounders/kinetic-ocaml-client"
+bug-reports: "https://github.com/cloudfounders/kinetic-ocaml-client"
+dev-repo: "https://github.com/cloudfounders/kinetic-ocaml-client"
```

Comment:
the manual is fuzzy (also contains a typo there)
> dev-repo: <string>: the URL of the package's source repository, which may be useful for developpers: not to be  > mistaken with the URL file, which points to the specific packaged version.

and opam lint accepts it. Anyway, I changed it.